stripe_test.go

```diff
@@ -68,6 +68,15 @@ func TestContext_Cancel(t *testing.T) {
 	assert.Regexp(t, regexp.MustCompile(`(request canceled|context canceled\z)`), err.Error())
 }
 
+func TestFormatURLPath(t *testing.T) {
+	assert.Equal(t, "/resources/1/subresources/2",
+		stripe.FormatURLPath("/resources/%s/subresources/%s", "1", "2"))
+
+	// Tests that each parameter is escaped for use in URLs
+	assert.Equal(t, "/resources/%25",
+		stripe.FormatURLPath("/resources/%s", "%"))
+}
```

Comment:
Okay, I added some tests (and an implementation) for this. See 78d772a.

Just a heads up though: this does add a little bit of runtime overhead because we have to check the response from `Sprintf` for bad values (see 78d772a for details there). There's also a minute chance it could cause a problem if a user has coupon or plan ID that contains `(MISSING)` or `%!(EXTRA `. That's hopefully quite unlikely, but still unfortunately possible.

Reply:
Gotcha, I did not think about the overhead. Maybe it's better to skip this then since it would mostly be our own mistake and it is unlikely to happen and not be caught by another crash or stripe-mock right?
Sorry for asking you to do the work and then undo it.